### PR TITLE
Devguide

### PR DESCRIPTION
**Pull request by itsNatTan**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ The `UI` component,
 * executes user commands using the `Logic` component.
 * listens for changes to `Model` data so that the UI can be updated with the modified data.
 * keeps a reference to the `Logic` component, because the `UI` relies on the `Logic` to execute commands.
-* depends on some classes in the `Model` component, as it displays `Person` object residing in the `Model`.
+* depends on some classes in the `Model` component, as it displays `Flashcard` object residing in the `Model`.
```
Comment (A1WAYSD) [inline docs/DeveloperGuide.md:80]: Good job for spotting this and adapting DG to our own project!

File: src/main/java/seedu/flashlingo/ui/FlashcardListPanel.java [hunk 2]
```diff
@@ -21,16 +21,13 @@ public class FlashcardListPanel extends UiPart<Region> {
...
     private ListView<FlashCard> flashcardListView;
-
-    private Model model;
     private MainWindow mw;
 
     /**
      * Creates a {@code PersonListPanel} with the given {@code ObservableList}.
      */
-    public FlashcardListPanel(ObservableList<FlashCard> flashcardList, Model model, MainWindow mw) {
+    public FlashcardListPanel(ObservableList<FlashCard> flashcardList, MainWindow mw) {
```
Comment (A1WAYSD) [inline src/main/java/seedu/flashlingo/ui/FlashcardListPanel.java:29]: It's great to see that you've removed unnecessary field!

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -289,103 +335,120 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 **Use case:** UC8 – Indicate user has forgotten word\
 **Actor:** User\
 **MSS:**
 1.	User indicates they couldn’t remember word.
-2.	Flashlingo displays motivational message to keep up.
+2. Flashlingo decements level of flashcard.
+3. Flashlingo displays motivational message to keep up.\
 Use case ends.
 
+**Extensions:**\
```
Comment (A1WAYSD) [inline docs/DeveloperGuide.md:417]: Good job on adding all the extensions! Seems like we should rewrite all MSS in details.

**Review by A1WAYSD — APPROVED** [review]

File: docs/images/UiClassDiagram.png
Comment (WangCheng0116) [inline docs/images/UiClassDiagram.png:1]: Good to see that you changed UML accordingly to Flashlingo Project

**Review by WangCheng0116 — APPROVED** [review]

**Pull request merged**